apps/editor/src/js/i18n/pt-br.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview I18N for Português
+ * @author Nícolas Huber <nicolasluishuber@gmail.com>
+ */
+import Editor from '../editor';
+
+Editor.setLanguage('pt', 'pt-BR', {
```

Comment:
@nicolashuber Oops... I did a build test on a local PC, but Portuguese wasn't set, so I checked again and the parameter type was wrong.

The ʻEditor.setLanguage` function can only take 2 parameters, and the first parameter must be a string or an array type. Check out the guide below.

> https://github.com/nhn/tui.editor/blob/master/apps/editor/docs/i18n.md#contributing